pkg/p2p/libp2p/libp2p.go

```diff
@@ -171,6 +172,34 @@ func New(ctx context.Context, signer beecrypto.Signer, networkID uint64, overlay
 		return nil, err
 	}
 
+	cfg := rcmgr.InfiniteLimits
+
+	in, out := 5_000, 10_000
```

Comment:
nitpick: maybe have these values as const; maybe global const?

Reply:
I would also suggest renaming them and stating what they can affect/tweak.

Reply:
fixed